Dockerfile

```diff
@@ -12,7 +12,7 @@
 ## limitations under the License.
 ## 
 
-FROM golang:1.23-bookworm AS builder
+FROM golang:1.24-bookworm AS builder
```

Comment:
Was this not picked up by dependabot?

Reply:
Same question. I even triggered Dependabot manually, and it didn't catch the bump